### PR TITLE
Fix incorrect wording for jdbc log filtering

### PR DESCRIPTION
**Pull request by Arshardh**
Fixes https://github.com/wso2/docs-apim/issues/1988

### CODE REVIEW

**Review by bhathiya — APPROVED** [review]

File: en/docs/administer/logging-and-monitoring/monitoring/working-with-observability.md [hunk 1]
```diff
@@ -774,4 +774,4 @@ In order to enable blacklisting of threads, add the following configuration as a
 
 If the above configuration is not added, by default the `MessageDeliveryTaskThreadPool` thread will be blacklisted as it is found to print a considerable amount for messages for API-M instances. However, if the above configuration is added the default value will be overridden. 
 
-Blacklisting of threads is not needed by default, as all unnecessary threads are already blacklisted.
\ No newline at end of file
+Blacklisting of threads is not needed by default, as all unnecessary threads are already blacklisted.
```
Comment (Mariangela) [inline en/docs/administer/logging-and-monitoring/monitoring/working-with-observability.md:777]: @Arshardh & @bhathiya , isn't it incorrect to use the term blacklisting in 3.2.0 docs?

**Comment by Mariangela** [comment]
@Arshardh , you need to send all new changes that go to 3.2.0 to the master branch as well.

**Comment by Arshardh** [comment]
I added a PR to the master branch as well 
https://github.com/wso2/docs-apim/pull/1991

File: en/docs/administer/logging-and-monitoring/monitoring/working-with-observability.md (hunk 1 above)
Comment (bhathiya) [reply en/docs/administer/logging-and-monitoring/monitoring/working-with-observability.md:777]: @Arshardh is this a config?

**Pull request merged**